Open Judge System/OJS.Common/GlobalConstants.cs

```diff
@@ -133,5 +137,9 @@ public static class GlobalConstants
         public static readonly string JavaSourceFileExtension = $".{CompilerType.Java.GetFileExtension()}";
         public static readonly string CSharpFileExtension = $".{CompilerType.CSharp.GetFileExtension()}";
         public static readonly string ClassDelimiter = $"~~!!!==#==!!!~~{Environment.NewLine}";
+
+        // Temp Directory folder paths
+        public static readonly string ExecutionStrategiesPath = 
```

Comment:
Misleading variable name. Is this the path where execution strategies reside? `ExecutionStrategiesWorkingDirectoryPath` or something similar would be better.